src/store/dtree.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable max-lines */
-import { cloneDeep } from 'lodash'
+import { cloneDeep, uniq } from 'lodash'
```

Comment:
better to
`import cloneDeep from 'lodash/cloneDeep'
`import uniq from 'lodash/uniq'`